alcs-frontend/src/app/services/application/application-reconsideration/application-reconsideration.service.ts

```diff
@@ -60,18 +47,19 @@ export class ApplicationReconsiderationService {
 
   async update(reconsiderationUuid: string, reconsideration: UpdateApplicationReconsiderationDto) {
     try {
-      await firstValueFrom(
+      return await firstValueFrom(
         this.http.patch<ApplicationReconsiderationDto>(`${this.url}/${reconsiderationUuid}`, reconsideration)
       );
     } catch (err) {
       console.error(err);
       this.toastService.showErrorToast('Failed to update reconsideration');
     }
+    return undefined;
```

Comment:
Just 'return;' for consistency with the rest of the code?

Reply:
Done!